Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker/RemoteDevicePicker.cs

```diff
@@ -0,0 +1,210 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Windows.System.RemoteSystems;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Picker Control to show List of Remote Devices that are accessible
+    /// </summary>
+    public sealed class RemoteDevicePicker : ContentDialog
+    {
+        private Dictionary<string, RemoteSystem> DeviceMap { get; set; }
+
+        private ListView _listDevices;
+        private ComboBox _listDeviceTypes;
+        private ProgressRing _progressRing;
+
+        /// <summary>
+        /// Gets List of All Remote Systems based on Selection Filter
+        /// </summary>
+        public ObservableCollection<RemoteSystem> RemoteSystems { get; private set; }
```

Comment:
Why does this need to be exposed publicly and why as an ObservableCollection rather than an ICollection?

Reply:
That doesn't change that this is exposed publicly and as an ObseravleCollection